editor/src/components/canvas/dom-lookup.ts

```diff
@@ -114,60 +85,72 @@ export function findFirstParentWithValidElementPathInner(
       }
     }
   }
+  return validStaticElementPaths
+}
+
+// Take a DOM element, and try to find the nearest selectable path for it
+export function findFirstParentWithValidElementPath(
```

Comment:
this also returns the element itself if that has the selectable path, right? so both the comment and the function name are a bit wrong, because it's nearest parent OR the element itself